packages/@react-spectrum/label/src/Field.tsx

```diff
@@ -35,7 +35,9 @@ function Field(props: SpectrumFieldProps, ref: Ref<HTMLElement>) {
     validationState,
     isInvalid,
     description,
-    errorMessage,
+    errorMessage = e => e.validationErrors.join(' '),
```

Comment:
ListFormat?

Reply:
hmm, well it's not a comma separated list or something like that. I assume the errors will be complete sentences on their own.